pkgs/gcc/gcc.yaml

```diff
@@ -12,6 +12,14 @@ sources:
   url: http://ftpmirror.gnu.org/gcc/gcc-4.9.2/gcc-4.9.2.tar.bz2
 
 build_stages:
+  - name: rpath_patch
+    before: configure
+    files: [rpath.patch]
+    handler: bash
+    bash: |
+      patch -up1 < _hashdist/rpath.patch
+      sed -i "s|@@ARTIFACT@@|${ARTIFACT}|g" gcc/config/i386/gnu-user.h gcc/config/i386/gnu-user64.h
```

Comment:
Right. The `${ARTIFACT}` path can contain both `:` and `|`, so to be completely safe, one would need to escape it, but it just makes the code really messy.